[TestResources] Parameter Addition and Doc Updates (#1973)
Azure/azure-sdk-tools: eng/common/TestResources/README.md

## On the Desktop

To set up your Azure account to run live tests, you'll need to log into Azure,
and create the resources defined in your `test-resources.json` or `test-resources.bicep` 
template as shown in the following example using Azure Key Vault. The script will create 

> a service principal automatically,or you may create a service principal you can save and 

Author: ```suggestion
a service principal automatically, or you may create a service principal you can save and 
```

> a service principal automatically,or you may create a service principal you can save and 
> reuse subsequently.

Author: ```suggestion
a service principal automatically, or you may create a service principal that can be reused subsequently.
```
